script/listen.rb

```diff
@@ -0,0 +1,11 @@
+# Simple test script to listen to UDP traffic.
```

Comment:
Should we stick a `#!/usr/bin/env ruby` shebang on this so it’s executable and doesn’t have to be invoked with `ruby` on the command line?